src/core/ledger/ledger.test.js

```diff
@@ -1416,9 +1416,12 @@ describe("core/ledger/ledger", () => {
         failsWithoutMutation(ledger, thunk, "invalid timestamp");
       }
     });
-  });
-
-  describe("timestamps", () => {
```

Comment:
nit: Looks like this was misnamed, not duplicate. Instead of bringing the non-timestamp-related `it` block under the `timestamps` describe block, maybe this describe block should be renamed `uuids` or something.

Reply:
Good catch!